Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java

```diff
@@ -104,9 +104,7 @@ private void logPurposeDecision(Boolean purposeFor, String hcid,
             purposeName = "PURPOSE OF";
         }
 
-        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid
-            + " and Service Name: " + serviceName + " is = "
+        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid + " and Service Name: " + serviceName + " is = "
```

Comment:
Sure!  I've been systematically working to replace concatenation with parameterized formatting, but some still slip through.  If I don't need another commit, I'll make a note for the next time we touch this class.